tests/test_confidence.py

```diff
@@ -0,0 +1,87 @@
+from vulture import Vulture
+
+from . import skip_if_python_2
+from . import skip_if_python_3
+
+
+def check_min_confidence(code, min_conf, linenos, confidences):
+    v = Vulture(verbose=True, min_confidence=min_conf)
+    v.scan(code)
+    for i, unused_item in enumerate(v.get_unused_code()):
```

Comment:
You can use zip: `for item, (lineno, confidence) in zip(v.get_unused_code(), tests):`